source/agora/common/Config.d

```diff
@@ -312,6 +315,7 @@ private NodeConfig parseNodeConfig (Node* node, const ref CommandLine cmdln)
     string data_dir = get!(string, "node", "data_dir")(cmdln, node);
     auto port = get!(ushort, "node", "port")(cmdln, node);
     auto validator_cycle = get!(uint, "node", "validator_cycle")(cmdln, node);
+    auto max_quorum_nodes = get!(uint, "node", "validator_cycle")(cmdln, node);
```

Comment:
Isn't it `max_quorum_nodes` instead of `validator_cycle`?

Reply:
typo! thanks for finding it.